translations/mgcp_rules.js

```diff
@@ -2336,7 +2337,8 @@ mgcp.rules = {
     // ##### End of dropList #####
 
     // Attribute names to swap on input. These are to avoid having copies of the same translation.
-    swapListIn : {'OHB':'HGT','CPYRT_NOTE':'CCN','SRC_INFO':'SDP','SRC_DATE':'SDV','SMC':'MCC'},
+    // swapListIn : {'OHB':'HGT','CPYRT_NOTE':'CCN','SRC_INFO':'SDP','SRC_DATE':'SDV','SMC':'MCC','LEN_':'LEN'},
+    swapListIn : {'OHB':'HGT','CPYRT_NOTE':'CCN','SRC_INFO':'SDP','SRC_DATE':'SDV','SMC':'MCC','LEN_':'LEN'},
```

Comment:
These two lines are the same, just one is commented out.  Please remove that one.
```suggestion
    swapListIn : {'OHB':'HGT','CPYRT_NOTE':'CCN','SRC_INFO':'SDP','SRC_DATE':'SDV','SMC':'MCC','LEN_':'LEN'},
```

Reply:
Done